packages/cli/src/api/executions.api.ts

```diff
@@ -38,6 +45,52 @@ import { getSharedWorkflowIds } from '../WorkflowHelpers';
 
 export const executionsController = express.Router();
 
+const schemaGetExecutionsQueryFilter = {
+	$id: '/IGetExecutionsQueryFilter',
+	type: 'object',
+	properties: {
+		finished: { type: 'boolean' },
+		mode: { type: 'string' },
+		retryOf: { type: 'string' },
+		retrySuccessId: { type: 'string' },
+		startedAt: {
+			type: { type: 'string' },
+			format: 'date-time',
+		},
+		stoppedAt: {
+			type: { type: 'string' },
+			format: 'date-time',
+		},
+		workflowData: { type: 'string' },
+		workflowId: { anyOf: [{ type: 'integer' }, { type: 'string' }] },
+	},
+};
+
+const allowedExecutionsQueryFilterFields = [
+	'finished',
+	'mode',
+	'retryOf',
+	'retrySuccessId',
+	'startedAt',
+	'stoppedAt',
+	'workflowData',
+	'workflowId',
```

Comment:
I left
```
	'finished',
	'mode',
	'retryOf',
	'retrySuccessId',
	'workflowId',
	'waitTill',
```
since they all seem potentially useful. Date filters in the current state are rather pointless since they'd have to match exactly on the millisecond..., so removed those as well.